src/controllers/trip.controller.js

```diff
@@ -175,6 +176,10 @@ class TripController {
   static async createComment(req, res) {
     const Id = req.params.tripRequestID;
     const data = await commentService.createComment(req, Id, 'trip request');
+    const tripRequest = await tripService.findRequestByID(db.requesttrip, { tripId: Id });
+    const tripRequestData = tripRequest[0].dataValues;
+    const receiver = (req.user.dataValues.id === tripRequestData.managerId) ? tripRequestData.userId : tripRequestData.managerId;
+    await NotificationService.sendNotification('trip_request_comment_event', receiver, 'New Comment', `${req.user.firstName} has posted a new comment`, data.dataValues.id, 'http://localhost:3000/api/v1/notifications');
```

Comment:
@shemaeric1234 , on this function, you should use the base url for us to be able to access it on the production link